DP21-JP/c100411014.lua

```diff
@@ -0,0 +1,53 @@
+--The Merciful Machine Angel
+--Logical Nonsense
+
+--Substitute ID
+local s,id=GetID()
+
+function s.initial_effect(c)
+	--Activate
+	local e1=Effect.CreateEffect(c)
+	e1:SetCategory(CATEGORY_DRAW)
+	e1:SetType(EFFECT_TYPE_ACTIVATE)
+	e1:SetProperty(EFFECT_FLAG_PLAYER_TARGET)
+	e1:SetCode(EVENT_FREE_CHAIN)
+	e1:SetCountLimit(1,id+EFFECT_COUNT_CODE_OATH)
+	e1:SetCost(s.cost)
+	e1:SetTarget(s.target)
+	e1:SetOperation(s.activate)
+	c:RegisterEffect(e1)
+end
+	--Check for "Cyber Angel" ritual monster
+function s.costfilter(c)
+	return (c:IsSetCard(0x2093) and c:IsType(TYPE_RITUAL)
```

Comment:
there a extra ( on there, it not necessary unless you missing something there

Reply:
Also should be c:IsRitualMonster - a new function that handles the busywork of distinguishing between ritual monster and ritual spell